### PR TITLE
[13.0] [ADD] product_category_description

### PR DESCRIPTION
**Pull request by MarcBForgeFlow**

### CODE REVIEW

File: product_category_description/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 ForgeFlow
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Category Description",
+    "summary": """
+        Allows to add a detailed description for a product category.""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow,Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline product_category_description/__manifest__.py]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

File: product_category_description/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Authors
+~~~~~~~
+
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* Denis Roussel <denis.roussel@acsone.eu>
```
Comment (MiquelRForgeFlow) [inline product_category_description/README.rst]: 😂

**Review by MiquelRForgeFlow — COMMENTED** [review]
please, run pre-commit :)

File: product_category_description/__manifest__.py (hunk 1 above)
Comment (MarcBForgeFlow) [reply product_category_description/__manifest__.py]: Done!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]

File: product_category_description/README.rst (hunk 2 above)
Comment (rousseldenis) [reply product_category_description/README.rst]: I'm honored by the copy-paste :smile:

**Pull request merged**